app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) == 3 and parts[0] == "cp":
+        source_file, destination_file = parts[1], parts[2]
```

Comment:
To enhance code clarity and avoid direct indexing, consider using unpacking and meaningful variable names